src/presentational-components/shared/table-empty-state.js

```diff
@@ -10,40 +10,53 @@ import {
   TextVariants
 } from '@patternfly/react-core';
 import { EmptyTable } from '@redhat-cloud-services/frontend-components/components/cjs/EmptyTable';
-import { useIntl } from 'react-intl';
 
-import tableEmptyMessages from '../../messages/table-empty.messages';
+const noRows = (title,
```

Comment:
Can this be converted to a react component? Plain functions make a bit of a mess in the virtual dom.

Reply:
will do